docs/DeveloperGuide.md

```diff
@@ -279,87 +279,87 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `SimplyKitchen` and the **Actor** is the `User`, unless specified otherwise)
+(For all use cases below, the **System** is the `SimplyKitchen` and the **Actor** is the `User`, unless otherwise specified)
```

Comment:
Perhaps can remove "**System** is the `SimplyKitchen`" as "System" is not used below?

Reply:
I believe the definition here is required, since use cases have System and Actor(s), here we say we are using SimplyKitchen to represent our System.